src/main/kotlin/com/atlassian/performance/tools/report/api/judge/Verdict.kt

```diff
@@ -14,20 +14,42 @@ class Verdict internal constructor(
     val reports: List<JUnitReport>,
     val failedActions: List<ActionType<*>>
 ) {
+    private companion object {
+        private val logger = LogManager.getLogger(this::class.java)
```

Comment:
Seems unused

Reply:
I didn't change the usage, just made it static.
However now I see private companion object 
```
 private companion object {
       // with or without @JvmStatic
        private val logger = LogManager.getLogger(this::class.java)
```
becomes
```
public final class Verdict {
   @Deprecated
   @NotNull
   public static final Companion Companion = new Companion((DefaultConstructorMarker)null);
```
I've also tested putting logger outside Verdict class

```
private val logger = LogManager.getLogger(Verdict::class.java)
class Verdict internal constructor(
```
Becomes
```
public final class VerdictKt {
   private static final Logger logger = LogManager.getLogger(Verdict.class);
   public static final Logger access$getLogger$p() {
      return logger;
   }
}
```

So it both cases it exposes public fields in the API. Is that something we can accept? I don't think anyone using this lib in Java would be interested in Companion object

Reply:
I think for now we should just use local `private val logger` inside the `Verdict` (without inner `object`), as this was our standard so far.

We could agree on different pattern which would optimise performance (one time logger initialization), however my suggestion would be to do it outside of this PR unless there is some specific concern around local logger of `Verdict`.